src/types.tsx

```diff
@@ -78,21 +78,10 @@ export interface FormikActions<Values> {
   /** Manually set values object  */
   setValues(values: Values): void;
   /** Set value of form field directly */
-  setFieldValue(
-    field: keyof Values,
```

Comment:
@weswigham I don't even think we retained type safety here because of the overload. Also AFAIK, can't really use `Extract` either because `setFieldValue` accepts dot path string like `social.facebook`.

Reply:
Given there's a `setFieldValue(field: string, value: any, shouldValidate?: boolean): void;` overload below it, I just assumed this overload existed to give completions for the known top-level fields.

Reply:
Oh you are totally right. I forgot that's what happened. 

So, just to be clear, `Extract<Values, string>` is not backwards compat?  hrmmm.

Reply:
You can say `keyof Values & string`